dataplane/src/pipeline/pipeline.rs

```diff
@@ -17,12 +17,27 @@ use crate::pipeline::{DynNetworkFunction, NetworkFunction, nf_dyn};
 #[derive(Default)]
 pub struct DynPipeline<Buf: PacketBufferMut> {
     nfs: Vec<Box<dyn DynNetworkFunction<Buf>>>,
+
+    #[allow(unused)]
+    name: String,
 }
 
+#[allow(unused)]
 impl<Buf: PacketBufferMut + 'static> DynPipeline<Buf> {
-    #[allow(unused)]
     pub fn new() -> Self {
-        Self { nfs: vec![] }
```

Comment:
Should we just require a name here?

Reply:
I added a with_name() constructor. Fine to remove it, but that forces us to specify the name in the new() ctr, or None if we want to make it optional.